middleware/expandDocument.js

```diff
@@ -0,0 +1,72 @@
+const dotProp = require('dot-prop');
```

Comment:
how does this module differ from the `_.get()` and `_.set()` functions provided in `lodash`?

Reply:
it's not. mistake on my part. fixed!